src/Microsoft.Docs.Template/Models/LandingData.cs

```diff
@@ -20,12 +20,9 @@ public class LandingData
         public LandingDataSection[] Sections { get; set; }
 
         public string DocumentType { get; set; }
-
-        [JsonExtensionData(WriteData = false)]
-        public JObject ExtensionData { get; set; }
     }
 
-    public sealed class LandingDataAbstract
+    public sealed class LandingDataAbstract : ClassWithExtensionData
```

Comment:
what if we just remove sealed?

Reply:
We need `ExtensionData` to store additional properties. `sealed` only affects `AdditaionalProperties` of the JSON schema while using `Json.net schema`, which we do not use any more.